lib/CHeroHandler.cpp

```diff
@@ -394,7 +401,7 @@ void CHeroClassHandler::afterLoadFinalization()
 		}
 	}
 
-	for(CHeroClass * hc : objects)
+	for(auto const & hc : objects)
```

Comment:
const auto. Or is that a pointer?

Reply:
Will change. Old habit :)
`const auto` and `auto const` are identical actually. (but not identical to `auto * const` in case of pointers)